aten/src/ATen/native/native_functions.yaml

```diff
@@ -8733,6 +8726,11 @@
 - func: max.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
 
+- func: max.unary_out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  device_check: NoCheck   # TensorIterator
+  dispatch:
+    CompositeExplicitAutograd: max_unary_out
```

Comment:
Just for my understanding: what's the motivation for adding this op?

One thing I can't tell from this is why we want to add an out= variant, but not a normal functional variant.

Reply:
oh nvm - it's so that `func: max` will get a proper out variant, right?

Reply:
Yeah.